### ISSUE TITLE
Tribler 7 failing after forced close.

### ISSUE THREAD

**Issue by imkebe** [Tribler/tribler#3347]
##### Tribler version/branch+revision: #####

7.0 rc 4

##### Operating system and version: #####

MacOS 10.13.2

##### Steps to reproduce the behavior: #####

I wanted to close Tribler. It didn't in 10 min so I forced quit. After relaunch it's failing.

```
./tribler
/Library/Python/2.7/site-packages/PyInstaller/loader/pyimod03_importers.py:546: RuntimeWarning: to-Python converter for boost::shared_ptr<libtorrent::alert> already registered; second conversion method ignored.
/Applications/Tribler.app/Contents/MacOS/matplotlib/font_manager.py:280: UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment.
INFO    1515704879.83        __init__:1627  __init__()
INFO    1515704880.29        __init__:1627  __init__(): returning
INFO    1515704880.29        __init__:1627  Shutting down Tribler
```

**Comment by xoriole** [comment]
@imkebe Thank you for reporting. We are investigating the issue with long closing time.

For the second part regarding failing to relaunch, there could be a zombie core process running in background. To find out : ```ps -ef | grep tribler``` or ```ps -ef | grep python```
If such a process exists, killing it and restarting tribler should work.
To kill a process : ```sudo kill -KILL <process_id>```
OR, simply restarting the OS should also work.

**Comment by imkebe** [comment]
@devos50 Thanks! There was a zombie process.

**Comment by devos50** [comment]
@imkebe thank you for your feedback. This issue will be addressed by @xoriole in #3349 (see also #3348).
